src/algokit/cli/goal.py

```diff
@@ -58,6 +58,13 @@ def goal_command(*, console: bool, goal_args: list[str]) -> None:
     compose_file_status = sandbox.compose_file_status()
     if compose_file_status is not ComposeFileStatus.UP_TO_DATE and sandbox.name == DEFAULT_NAME:
         raise click.ClickException("LocalNet definition is out of date; please run `algokit localnet reset` first!")
+    ps_result = sandbox.ps("algod")
+    match ps_result:
+        case [{"State": "running"}]:
+            pass
+        case _:
+            logger.info("AlgoKit LocalNet isn't running")
```

Comment:
```suggestion
            logger.info("LocalNet isn't running. Starting LocalNet...")
```

Reply:
I feel it might be repetitive to add starting Localnet twice.
<img width="618" alt="Screenshot 2024-01-09 at 12 13 37 am" src="https://github.com/algorandfoundation/algokit-cli/assets/34777331/f287c7c6-d4ee-4148-90a3-6f6192971af5">

Reply:
Oh my mistake, didn't realise the `Starting AlgoKit LocaNet now...` was printed.
You can just go with `LocalNet isn't running`, I've updated my suggestion.